packages/blocks/src/__internal__/service/legacy-services/note-service.ts

```diff
@@ -4,6 +4,7 @@ import type { NoteBlockModel } from '../../../note-block/note-model.js';
 import type { SerializedBlock } from '../../utils/index.js';
 import { addSerializedBlocks } from '../json2block.js';
 import { BaseService } from '../service.js';
+import { getService } from '../singleton.js';
```

Comment:
@zuoxiaodong0815 the workaround here is sourced from your `// @ts-ignore`, the parameter doesn't match. This should be fixed.